target/linux/lantiq/image/ar9.mk

```diff
@@ -29,6 +29,21 @@ define Device/avm_fritz7320
 endef
 TARGET_DEVICES += avm_fritz7320
 
+define Device/avm_fritz7330
+  $(Device/AVM_preloader)
+  DEVICE_MODEL := FRITZ!Box 7330
+  SOC := ar9
+  IMAGE_SIZE := 15744k
+  LOADER_FLASH_OFFS := 0x31000
+  DEVICE_PACKAGES := kmod-ath9k kmod-owl-loader wpad-basic-mbedtls \
```

Comment:
The DTS file and these settings are the same except the name of the board. I would prefer to just add an  alt entry to the fritzbox 7320 entry. 
````
DEVICE_ALT1_VENDOR :=AVM 
DEVICE_ALT1_MODEL := FRITZ!Box 7330
````

Reply:
You are right that the dts for both devices are same. I expect they will differ in the future. I have currently followed the suggested approach and only added an alternative name for the AVM 7330. The dts files for both devices can be spliced in the future when necessary